app/components/Map.js

```diff
@@ -181,7 +181,7 @@ export default class Map extends Component {
         <MapView
           ref="map"
           showsUserLocation={true}
-          initialRegion={{ longitudeDelta: 0.005, latitude: currLoc.latitude,longitude: currLoc.longitude, latitudeDelta: 0.005 }}
+-         initialRegion={{ longitudeDelta: 0.005, latitude: currLoc.latitude,longitude: currLoc.longitude, latitudeDelta: 0.005 }}
```

Comment:
Also having this line throws a warning, but that is because of a bug a previous PR introduced wherein 'currLoc' is not the object we would expect because it is now a firebase ref.

Best to just remove this line for now.